tasmanian/CMakeLists.txt

```diff
@@ -0,0 +1,6 @@
+find_package(Tasmanian 7.5 PATHS "${TASMANIAN_DIR}" REQUIRED)
+
+add_executable(tasmanian_magma example_unstructured_grid.cpp)
+target_link_libraries(tasmanian_magma Tasmanian::Tasmanian)
+
+add_test(tasmanian tasmanian_magma)
```

Comment:
Please use the [xsdk_add_test macro](https://github.com/xsdk-project/xsdk-examples/blob/master/cmake/XsdkAddTest.cmake).

Reply:
Example: https://github.com/xsdk-project/xsdk-examples/blob/master/sundials/CMakeLists.txt#L41

Reply:
@mkstoyanov I think this is the last thing that needs to be addresed.

Reply:
Should be good now.